cache/test_purge_resp_hch.py

```diff
@@ -0,0 +1,179 @@
+from framework import tester
+from helpers import tf_cfg, deproxy, tempesta
+import copy
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2022 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+class HeavyChunkedPurgeRespTest(tester.TempestaTest):
+    # This is another heavy chunked test for ss_skb_chop_head_tail() function
+    # in context of rewriting PURGE method as GET, issue #1535, now testing
+    # with chunked response
+    #
+    backends_template = [
+        {
+            'id' : 'deproxy',
+            'type' : 'deproxy',
+            'port' : '8000',
+            'keep_original_data' : True,
+            'response' : 'static',
+            'response_content' :
+"""HTTP/1.1 200 OK
+Content-Length: 8
+Content-Type: text/plain
+Connection: keep-alive
+
+THE PAGE
+"""
+        },
+    ]
+
+    tempesta = {
+        'config' : """
+cache 2;
+server ${general_ip}:8000;
+cache_fulfill * *;
+cache_methods GET HEAD;
+cache_purge;
+cache_purge_acl ${client_ip};
+
+""",
+    }
+
+    clients = [
+        {
+            'id' : 'deproxy',
+            'type' : 'deproxy',
+            'addr' : "${tempesta_ip}",
+            'port' : '80'
+        },
+    ]
+
+    BODY_LENGTH = 65536
+
+    def setUp(self):
+        self.backends = copy.deepcopy(self.backends_template)
+        self.backends[0]['response_content'] = self.generate_content()
+        super(HeavyChunkedPurgeRespTest, self).setUp()
+
+    def generate_content(self):
+        body = "x"*self.BODY_LENGTH
+        return (
+"""HTTP/1.1 200 OK
+Content-Length: %d
+Content-Type: text/plain
+Connection: keep-alive
+
+%s
+""" % (self.BODY_LENGTH, body)
+        )
+
+    def common_check(self, chunksize=0, request_0='', expect_status_0=200,
+                           request='', expect_status=200, expect='',
+                           expect_body=False):
+        # Set expect to expected proxied request,
+        # to empty string to skip request check and
+        # to None to check that request is missing
+        deproxy_srv = self.get_server('deproxy')
+        deproxy_srv.segment_size = chunksize
+        #print (deproxy_srv.response)
```

Comment:
There is still a commented out `print()` :) Why don't you use the standard debug printing from the framework?

Reply:
Print produces more focused informaion than huge debug printing from framework :)
This print is not needed more, I'll delete it.